ouroboros-consensus/src/Ouroboros/Consensus/Node/ProtocolInfo/Byron.hs

```diff
@@ -139,6 +138,8 @@ protocolInfoByron (NumCoreNodes numCoreNodes) params gc mLeader =
   where
     delegation :: Cardano.Genesis.GenesisDelegation
     delegation = Cardano.Genesis.configHeavyDelegation gc
+    numCoreNodes = Set.size . Cardano.Genesis.unGenesisKeyHashes $
+                   Cardano.Genesis.configGenesisKeyHashes gc
```

Comment:
:+1:  Though note that we'll need the `genesisKeyHashes` as a `Set` on it's own for the `genesisKeyCoreNodeId` impl. So this will become
```
numCoreNodes = Set.size genesisKeyHashes
```

Reply:
But I've factored `genesisKeyCoreNodeId` into PBFT, as per TODO..